newIDE/app/src/UI/TreeView/index.js

```diff
@@ -401,6 +408,22 @@ const TreeView = <Item: ItemBaseAttributes>(
     [openedNodeIds]
   );
 
+  const animateItem = React.useCallback(
+    (item: Item) => {
+      setAnimatedItemId(getItemId(item));
+    },
+    [getItemId]
+  );
+
+  React.useEffect(
+    () => {
+      if (animatedItemId) {
+        setTimeout(() => setAnimatedItemId(''), 400);
```

Comment:
In theory you should cancel the timeout if the animatedItemId is changed, so that one can:

- Animate item 1
- 200ms after, Animate item 2 (item 1 stops animating)
- 200ms after, the animated item id is NOT reset (because it's the timeout of the first animatedItemId)
- 200ms after, the animated item id IS reset (it's the second timeout, and it's perfectly fine and does not need to be cancelled).

Reply:
I was lazy and thought that 400ms were too quick for anything like this to happen 😅 
It's now fixed